packages/components/src/standalone-comparison-grid/style.scss

```
@@ -0,0 +1,64 @@
+// ==========================================================================
+// Comparison Grid
+// ==========================================================================
+@import "../styles/typography.scss";
+
+.standalone-comparison-grid {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+	gap: 20px;
+	padding: 20px;
+
+	.standalone-comparison-grid__column {
+		display: flex;
+		flex-direction: column;
+		gap: 10px;
+	}
+
+	.standalone-comparison-grid__body {
```

```suggestion
	.&_body {
```

The `&`  will repeat the name of the parent classname